src/audio_unit/mod.rs

```diff
@@ -1,4 +1,5 @@
-mod delay;
+pub mod delay;
```

Comment:
Why is this now `pub` vs the other units still not being `pub`?

Reply:
To expose the `DelayMs` type in `delay`. I could also have `pub use`d that, I guess.